cmd/wigMath/wigMath_test.go

```diff
@@ -0,0 +1,228 @@
+		MinValue:               -1 * math.MaxFloat64,
+		MaxValue:               math.MaxFloat64,
+		ScalarMultiply:         1,
+		ScalarDivide:           1,
+		AddFile:                "",
+		SubtractFile:           "testdata/second.wig",
+		MovingAverageSmoothing: 1,
+		AbsoluteError:          "",
+		AbsolutePercentError:   "",
+		Missing:                -10,
+		Pearson:                "",
+		SamplingFrequency:      1,
+	},
+	{
+		InFile:                 "testdata/unsmooth.wig",
+		OutFile:                "testdata/tmp.smooth.wig",
+		ExpectedFile:           "testdata/expected.smooth.wig",
+		MinValue:               -1 * math.MaxFloat64,
+		MaxValue:               math.MaxFloat64,
+		ScalarMultiply:         1,
+		ScalarDivide:           1,
+		AddFile:                "",
+		SubtractFile:           "",
+		MovingAverageSmoothing: 5,
+		AbsoluteError:          "",
+		AbsolutePercentError:   "",
+		Missing:                -10,
+		Pearson:                "",
+		SamplingFrequency:      1,
+	},
+	{
+		InFile:                 "testdata/in.wig",
+		OutFile:                "testdata/tmp.absError.wig",
+		ExpectedFile:           "testdata/expected.absError.wig",
+		MinValue:               -1 * math.MaxFloat64,
+		MaxValue:               math.MaxFloat64,
+		ScalarMultiply:         1,
+		ScalarDivide:           1,
+		AddFile:                "",
+		SubtractFile:           "",
+		MovingAverageSmoothing: 1,
+		AbsoluteError:          "testdata/second.wig",
+		AbsolutePercentError:   "",
+		Missing:                -10,
+		Pearson:                "",
+		SamplingFrequency:      1,
+	},
+	{
+		InFile:                 "testdata/in.wig",
+		OutFile:                "testdata/tmp.absPercentError.wig",
+		ExpectedFile:           "testdata/expected.absPercentError.wig",
+		MinValue:               -1 * math.MaxFloat64,
+		MaxValue:               math.MaxFloat64,
+		ScalarMultiply:         1,
+		ScalarDivide:           1,
+		AddFile:                "",
+		SubtractFile:           "",
+		MovingAverageSmoothing: 1,
+		AbsoluteError:          "",
+		AbsolutePercentError:   "testdata/second.wig",
+		Missing:                -10,
+		Pearson:                "",
+		SamplingFrequency:      1,
+	},
+	{
+		InFile:                 "testdata/in.wig",
+		OutFile:                "testdata/tmp.Pearson.txt",
+		ExpectedFile:           "testdata/expected.Pearson.txt",
+		MinValue:               -1 * math.MaxFloat64,
+		MaxValue:               math.MaxFloat64,
+		ScalarMultiply:         1,
+		ScalarDivide:           1,
+		AddFile:                "",
+		SubtractFile:           "",
+		MovingAverageSmoothing: 1,
+		AbsoluteError:          "",
+		AbsolutePercentError:   "",
+		Missing:                -10,
+		Pearson:                "testdata/second.wig",
+		SamplingFrequency:      1,
+	},
+	{
+		InFile:                 "testdata/in.wig",
+		OutFile:                "testdata/tmp.Mult55.wig",
+		ExpectedFile:           "testdata/expected.Mult55.wig",
```

Comment:
The actual file name is testdata/expected.mult55.wig so the capitalization of "M" here caused a mismatch and that's why go test can't find it